cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -25,108 +26,329 @@
+                                            conversion_tags,
+                                            form_data_vm_obj_single_datastore):
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+
+    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore[0])
+    request.addfinalizer(lambda: infrastructure_mapping_collection.delete(mapping))
+
+    migration_plan_collection = appliance.collections.v2v_plans
+
+    migration_plan = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()), description="desc_{}"
+        .format(fauxfactory.gen_alphanumeric()), infra_map=mapping.name,
+        vm_list=form_data_vm_obj_single_datastore[1], start_migration=True)
+
+    # explicit wait for spinner of in-progress status card
+    view = appliance.browser.create_view(navigator.get_class(migration_plan_collection, 'All').VIEW)
+    wait_for(func=view.progress_card.is_plan_started, func_args=[migration_plan.name],
+        message="migration plan is starting, be patient please", delay=5, num_sec=150,
+        handle_exception=True)
+
+    def _get_plan_status():
+        is_plan_visible = view.progress_card.is_plan_visible(migration_plan.name)
+        if is_plan_visible:
+            # log current status
+            # uncomment following logs if github.com/ManageIQ/miq_v2v_ui_plugin/issues/415 fixed
+            # logger.info("For plan %s, current migrated size is %s out of total size %s",
+            #     migration_plan.name, view.progress_card.get_migrated_size(migration_plan.name),
+            #     view.progress_card.get_total_size(migration_plan.name))
+            # logger.info("For plan %s, current migrated VMs are %s out of total VMs %s",
+            #     migration_plan.name, view.progress_card.migrated_vms(migration_plan.name),
+            #     view.progress_card.total_vm_to_be_migrated(migration_plan.name))
+            logger.info("For plan %s, is plan in progress: %s, time elapsed for migration: %s",
+                migration_plan.name, view.progress_card.is_plan_visible(migration_plan.name),
+                view.progress_card.get_clock(migration_plan.name))
+        # return False if plan visible under "In Progress Plans"
+        return not is_plan_visible
 
-@pytest.fixture(scope="module")
-def vm_list(request, appliance, second_provider, provider):
-    """Fixture to provide list of vm objects"""
-    # TODO: Need to add list of vm and its configuration in cfme_data.yaml
-    templates = [second_provider.data.templates.big_template['name']]
-    for template in templates:
-        vm_name = random_vm_name(context='v2v-auto')
-        collection = appliance.provider_based_collection(second_provider)
-        vm = collection.instantiate(vm_name, second_provider, template_name=template)
+    # wait until plan is in progress
+    wait_for(func=_get_plan_status, message="migration plan is in progress, be patient please",
```

Comment:
So this is hitting two different failures in PRT.

1. DOM no longer attached, I think we need some refresh in here
2. The plan completes, and the page is still on the 'In progress plans' only, so the plan is no longer visible.

The plan that we're looking for is visible in the PRT screenshot in the notification tray, but isn't listed in the plans table.
This needs to either use the migr-dropdown to make the plan visible in the table, or look in the notification tray.

The MIQ V2V UI is going through some refactoring as you've pointed out, so please capture the expected behavior and plans for dealing with it into comments here.  In the meantime, the test needs some stability to handle this behavior.